webui/src/components/Files.jsx

```diff
@@ -93,11 +131,17 @@ class Files extends React.Component {
                 postUrl: getApiURL()
             };
             return <div>
-                <DropzoneComponent config={config} eventHandlers={eventHandlers} djsConfig={djsConfig} />
+                <span className={(sessionStorage.getItem("buildID")) ? "dz-hidden" : "dz-visible"}>
```

Comment:
better to just use the ternary operator here, instead of classes, for showing the span